client/build/asset-manifest.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
build folder should be ignored from git, in .gitignore switch `/build/` to `*/build/`